src/compose/images.ts

```diff
@@ -347,7 +347,7 @@ export class Images extends (EventEmitter as new () => ImageEventEmitter) {
 	}
 
 	private async getImagesForCleanup(): Promise<string[]> {
-		const images = [];
+		const images: string[] = [];
```

Comment:
All these changes to typescript files for vars that don't have an initial type are because they fail under `noImplicitAny: false`